.github/workflows/wheel.yml

```diff
@@ -0,0 +1,100 @@
+name: wheels
+on:
+  pull_request:
+  push:
+    branches: [master, test-me-*]
+    tags: ['*']
+
+jobs:
+  manylinux:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+        - name: cp27-cp27mu
+          image: quay.io/pypa/manylinux1_x86_64
+        - name: cp27-cp27m
+          image: quay.io/pypa/manylinux1_x86_64
+        - name: cp35-cp35m
+          image: quay.io/pypa/manylinux1_x86_64
+        - name: pp27-pypy_73
+          image: pypywheels/manylinux2010-pypy_x86_64
+        - name: pp36-pypy36_pp73
+          image: pypywheels/manylinux2010-pypy_x86_64
+        - name: pp37-pypy37_pp73
+          image: pypywheels/manylinux2010-pypy_x86_64
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: recursive
+    - run: |
+        mkdir dist
+        podman run --rm \
+          -v $PWD:/src:ro \
+          -v $PWD/dist:/dist:rw \
+          ${{ matrix.image }} \
+          bash -exc '\
+            yum install -y libffi-devel && \
+            /opt/python/${{ matrix.name }}/bin/pip wheel \
+              --no-deps --wheel-dir /tmp/wheels /src && \
+            auditwheel repair /tmp/wheels/* --wheel-dir /dist && \
+            find /dist -type f | xargs --verbose --replace unzip -l {} \
+          '
+      name: build wheel
+    - uses: actions/upload-artifact@v2
+      with:
+        name: wheels-manylinux-${{ matrix.name }}
+        path: dist/*.whl
+  macos:
+    runs-on: macos-latest
+    strategy:
+      matrix:
+        include:
+        - name: cp27
+          py: 2.7
+        - name: cp35
+          py: 3.5
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: recursive
+    - uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.py }}
+    - run: python -m pip wheel --no-deps --wheel-dir dist .
+    - uses: actions/upload-artifact@v2
+      with:
+        name: wheels-macos-${{ matrix.name }}
+        path: dist/*.whl
+  windows:
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        include:
+        - name: cp27-x86
+          py: 2.7
+          arch: x86
+        - name: cp27-x64
+          py: 2.7
+          arch: x64
+        - name: cp35-x86
+          py: 3.5
+          arch: x86
+        - name: cp35-x64
```

Comment:
Are you positive Windows will work with just `cp35` on 3.6, 3.7 and 3.8? I was building [for all of them](https://pypi.org/project/argon2-cffi/#files), I thought the ABI3 stuff would require a release (i.e. 3.9)? 🤔

Reply:
yep!  that's what #69 does -- it's the same as the stuff that's already in place for linux / macos in the current release:

```
C:\Users\asott\AppData\Local\Temp>unzip C:\Users\asott\Desktop\wheels-windows-cp35-x64.zip
Archive:  C:\Users\asott\Desktop\wheels-windows-cp35-x64.zip
  inflating: argon2_cffi-20.2.0.dev0-cp35-abi3-win_amd64.whl

C:\Users\asott\AppData\Local\Temp>C:\python38\python -m venv venv

C:\Users\asott\AppData\Local\Temp>venv\Scripts\pip --version
pip 20.1.1 from c:\users\asott\appdata\local\temp\venv\lib\site-packages\pip (python 3.8)

C:\Users\asott\AppData\Local\Temp>venv\Scripts\pip install argon2_cffi-20.2.0.dev0-cp35-abi3-win_amd64.whl
Processing c:\users\asott\appdata\local\temp\argon2_cffi-20.2.0.dev0-cp35-abi3-win_amd64.whl
Collecting cffi>=1.0.0
  Downloading cffi-1.14.3-cp38-cp38-win_amd64.whl (179 kB)
     |████████████████████████████████| 179 kB 2.2 MB/s
Collecting six
  Using cached six-1.15.0-py2.py3-none-any.whl (10 kB)
Collecting pycparser
  Using cached pycparser-2.20-py2.py3-none-any.whl (112 kB)
Installing collected packages: pycparser, cffi, six, argon2-cffi
Successfully installed argon2-cffi-20.2.0.dev0 cffi-1.14.3 pycparser-2.20 six-1.15.0
WARNING: You are using pip version 20.1.1; however, version 20.2.4 is available.
You should consider upgrading via the 'c:\users\asott\appdata\local\temp\venv\scripts\python.exe -m pip install --upgrade pip' command.

C:\Users\asott\AppData\Local\Temp>venv\Scripts\python
Python 3.8.5 (tags/v3.8.5:580fbb0, Jul 20 2020, 15:57:54) [MSC v.1924 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import argon2
>>> argon2.verify_password(argon2.hash_password(b'secret', b'somesalt'), b'secret')
True
```

Reply:
I was trying to test it myself but I'm too dumb for Windows so I'll just take your word for it. 😅